tests/api/utils/test_builder.py

```diff
@@ -420,6 +421,7 @@ def test_resolve_mlrun_install_command_version():
 
 def test_build_runtime_ecr_with_ec2_iam_policy(monkeypatch):
     _patch_k8s_helper(monkeypatch)
+    _mock_import_hub_function(monkeypatch)
```

Comment:
this is not related to the test, just change the `set_function` to not import a hub function